Code/Tools/SceneAPI/SceneCore/Tests/Utilities/PatternMatcherTests.cpp

```diff
@@ -56,6 +68,23 @@ namespace AZ
                 PatternMatcher matcher("^.{4}$", PatternMatcher::MatchApproach::Regex);
                 EXPECT_FALSE(matcher.MatchesPattern("string_to_long_for_regex"));
             }
+
+            TEST(PatternMatcherTest, MatchesPattern_MatchingPrefixInArrayOfPatterns_ReturnsTrue)
+            {
+                AZStd::array<AZStd::string_view, 3> patterns = { "postfix", "xxx", "prefix_" };
+
+                PatternMatcher matcher(patterns, PatternMatcher::MatchApproach::PreFix);
+                EXPECT_TRUE(matcher.MatchesPattern("prefix_for_string"));
+            }
+
+            TEST(PatternMatcherTest, MatchesPattern_NonMatchingPrefixInArrayOfPatterns_ReturnsFalse)
+            {
+                AZStd::array<AZStd::string_view, 2> patterns = { "postfix", "xxx" };
```

Comment:
Repeat: This can be constexpr as well
```suggestion
                constexpr auto patterns = AZStd::to_array<AZStd::string_view>({ "postfix", "xxx" });
```